### DISCUSSION TITLE
Fading parameters for EnvShake

### DISCUSSION THREAD

**Discussion by potsmugen** [ikemen-engine/Ikemen-GO#1036] [Feature Request]
### Is your feature request related to a problem? Please describe.

If you for instance play Street Fighter Alpha and perform Akuma's taunt, you will see that the EnvShake effect in that game fades out. The screen shaking amplitude gradually decreases until the screen stops shaking.

Currently you have to recreate this effect manually (see my Shin Gouki's taunt for example).

### Describe the solution you'd like

I propose a new "mul" (or something) parameter for EnvShake (both on its own and in Hitdefs). For every EnvShake cycle, the amplitude is multiplied by this value.

For example:

```
[State 3051, Shake]
type = envshake
trigger1 = !time
time = 30
ampl = 5
freq = 180
mul = 0.9
```

In this case, the screen shakes 5 pixels in the first cycle, then 4.50 pixels, then 4.05 and so on.

This parameter would default to 1, of course.

**Comment by K4thos** [comment]
implemented: https://github.com/ikemen-engine/Ikemen-GO/pull/1081
